packages/trader/src/Modules/Trading/Components/Form/screen-small.jsx

```diff
@@ -66,6 +67,11 @@ const CollapsibleTradeParams = ({
                     <RiskManagementInfo />
                 </div>
             )}
+            {is_turbos && (
+                <div collapsible='true'>
+                    <RiskManagementInfo />
+                </div>
+            )}
```

Comment:
```suggestion

```
@kate-deriv instead, i suggest adding `is_turbos &&` to line 65 like this 🙂:
{is_multiplier && is_turbos && (

Reply:
Great idea, please excuse my inattention

Reply:
But may be we should add {is_multiplier || is_turbos && ...} as they can't be both equal to true at the same time

Reply:
yeah, sorry, i meant `||` 😅